tests/app/db.py

```diff
@@ -660,3 +662,44 @@ def create_service_data_retention(
         days_of_retention=days_of_retention
     )
     return data_retention
+
+
+def create_sample_inbound_numbers(service=None):
+    if not service:
+        service = create_service(service_name='sample service 2')
+    service_2 = create_service(service_name='service 2 for inbound')
+    inbound_numbers = list()
+    inbound_numbers.append(create_inbound_number(number='1', provider='mmg'))
+    inbound_numbers.append(create_inbound_number(number='2', provider='mmg', active=False, service_id=service.id))
+    inbound_numbers.append(create_inbound_number(number='3', provider='firetext',
+                                                 service_id=service_2.id))
+    return inbound_numbers
+
+
+def create_code(code_type, usr=None, code=None):
```

Comment:
I'm not sure we need this function - we're calling it twice - passing in a `usr` and a `code`. I think it would be easier to read just calling `create_user_code` from the calling function and not have this function until we know there's an need for it